README.md

```diff
@@ -104,6 +105,12 @@ if(!isWholeCodeCoveraged) return
 - Keep the current tests and make them pass! 
 Once the current code is tested and can be refactored. We must make sure that the new changes will not break the current tests. 
 
+### 1.3 Imports
+
+If the data to be imported belongs to `another module/scope`, use an **absolute path**.
+
+If the data to be imported belongs to the `same module/scope`, use **relative path**.
```

Comment:
You could add some example. What do you think?

Reply:
[Done](https://github.com/juntossomosmais/frontend-guideline/pull/33/commits/331f5c371a789fdef18f3668a72c642551df1040)